perfkitbenchmarker/network.py

```diff
@@ -51,12 +51,14 @@ def GetFirewall(cls):
         benchmark_spec.firewalls[key] = cls()
       return benchmark_spec.firewalls[key]
 
-  def AllowPort(self, vm, port):
+  def AllowPort(self, vm, start_port, end_port=None):
```

Comment:
In this and a couple other places, you seem to have the `start_port` and `end_port` changes that you already submitted in a different PR. Merging the master branch into your branch should fix this.